lib/elsa/consumer/initializer.ex

```diff
@@ -23,6 +23,8 @@ defmodule Elsa.Consumer.Initializer do
   end
 
   defp configure_topic(topic, registry, brod_client, config) do
+    :brod_client.get_metadata(brod_client, topic)
```

Comment:
Wrap this in a private function to better communicate that it updates global state from `brod`?